### PR TITLE
feat: touch up logging

### PR DESCRIPTION
**Pull request by bschaatsbergen**
- Created a FlagStore struct to store flags on
- Added a 'downloads' SVG badge to the README
- Added logrus for structured logging
- Added debug logging
- Added color to the `--help` / usage
- lower cased the constants to be in-line with Go naming convention

image

### CODE REVIEW

File: pkg/model/flagstore.go [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package model
+
+type Flagsore struct {
```
Comment (mmourick) [inline pkg/model/flagstore.go:3]:
```suggestion
type Flagstore struct {
```

File: cmd/root.go [hunk 2]
```diff
@@ -2,24 +2,32 @@ package cmd
...
 )
 
+type PlainFormatter struct{}
+
 var (
-	version                string
-	dnsServerIP            string
-	userSpecifiedQueryType string
+	version   string
+	flagStore model.Flagsore
```
Comment (mmourick) [inline cmd/root.go:20]:
```suggestion
	flagStore model.Flagstore
```

**Review by mmourick — APPROVED** [review]
Nice additions! Left some small comments, good to merge after that.

File: cmd/root.go (hunk 2 above)
Comment (bschaatsbergen) [reply cmd/root.go]: Fixed

File: pkg/model/flagstore.go (hunk 1 above)
Comment (bschaatsbergen) [reply pkg/model/flagstore.go]: Fixed

**Comment by bschaatsbergen** [comment]
addressed the comments @mmourick

**Review by mmourick — APPROVED** [review]
Looks good to me!

**Pull request merged**